### PR TITLE
Add PyROOT THistPainter documentation for keeping the TBrowser open

### PR DESCRIPTION
**Pull request by skkwan**
# This Pull request:
Creates a Pythonization file `ROOT/_pythonization/_thistpainter.py` which only contains documentation on how to keep the TBrowser open in several simple PyROOT examples. Based on interaction/information in this [forum post](https://root-forum.cern.ch/t/oop-pyroot-histogram-plotted-from-file-disappears-on-click/59303).

Please let me know if there is anything to fix/change.

## Changes or fixes:

Addresses hackathon item ["Document TH1 drawing with Python functions"](https://github.com/orgs/root-project/projects/18?pane=issue&itemId=88465159). 

## Checklist:

- [x] tested changes locally
- [x] updated the docs (if necessary)

### CODE REVIEW

File: bindings/pyroot/pythonizations/python/ROOT/_pythonization/_thistpainter.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        '''
+        test_histo = ROOT.TH1D("test_histo", "Histogram to draw", 200, -5, 5)
+        test_histo.FillRandom("gaus", 1000)
+
+        test_canv = ROOT.TCanvas("test_canv", "test_canv", 900, 700)
+        ROOT.SetOwnership(test_canv, False)
+        ROOT.SetOwnership(test_histo, False)
+        test_canv.SaveAs("myDrawExample.png")
+
+    if __name__ == '__main__':
```
Comment (vepadulano) [inline bindings/pyroot/pythonizations/python/ROOT/_pythonization/_thistpainter.py:51]: Similar comment about the indentation level

File: bindings/pyroot/pythonizations/python/ROOT/_pythonization/_thistpainter.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    test_histo.Draw()
+    \endcode
+
+    It is sufficient to call Python with the flag `-i` to keep the TBrowser open:
+    \code{.sh}
+    # -i flag keeps the TBrowser open with the TCanvas on it
+    python -i makeAndDrawHisto.py
+    \endcode
+
+- If the call to TH1::Draw is not at top-level, both the TCanvas and TH1 objects need to remain in scope. One way to accomplish this is with `SetOwnership`, as in this example:
```
Comment (vepadulano) [inline bindings/pyroot/pythonizations/python/ROOT/_pythonization/_thistpainter.py:36]:
```suggestion
- If the call to TH1.Draw is not at top-level, both the TCanvas and TH1 objects need to remain in scope. One way to accomplish this is with `SetOwnership`, as in this example:
```

Comment (vepadulano) [inline bindings/pyroot/pythonizations/python/ROOT/_pythonization/_thistpainter.py:36]:
```suggestion
- If the call to TH1::Draw is not at top-level, both the TCanvas and TH1 objects need to remain in scope. One way to accomplish this is with ROOT.SetOwnership, as in this example:
```

File: bindings/pyroot/pythonizations/python/ROOT/_pythonization/_thistpainter.py [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+        test_canv.SaveAs("myDrawExample.png")
+
+    if __name__ == '__main__':
+        main()
+    \endcode
+
+    To keep the TBrowser open, the Python script should be run with the `-i` flag:
+    \code{.sh}
+    python -i makeAndDrawHistoInMain.py
+    \endcode
```
Comment (vepadulano) [inline bindings/pyroot/pythonizations/python/ROOT/_pythonization/_thistpainter.py:58]: Similar comment about the indentation level

File: bindings/pyroot/pythonizations/python/ROOT/_pythonization/_thistpainter.py [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+\htmlonly
+</div>
+</details>
+\endhtmlonly
+*/
```
Comment (vepadulano) [inline bindings/pyroot/pythonizations/python/ROOT/_pythonization/_thistpainter.py:64]: Remove these lines in favour of `\endpythondoc`, see example at  https://github.com/root-project/root/blob/e8aadfc845c3e98ba88e6893eae8832f8fd2296e/bindings/pyroot/pythonizations/python/ROOT/_pythonization/_tfile.py#L63

File: bindings/pyroot/pythonizations/python/ROOT/_pythonization/_thistpainter.py [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+
+r"""
+/**
+\class THistPainter
+\brief \parblock \endparblock
+\htmlonly
+<details open>
+<summary  style="font-size:20px; color: #425788;"><b>Python interface</b></summary>
+<div class="pyrootbox">
+\endhtmlonly
```
Comment (vepadulano) [inline bindings/pyroot/pythonizations/python/ROOT/_pythonization/_thistpainter.py:18]: We have recently merged changes that introduce a macro which makes this code redundant, use `\pythondoc THistPainter`, example shown at https://github.com/root-project/root/blob/e8aadfc845c3e98ba88e6893eae8832f8fd2296e/bindings/pyroot/pythonizations/python/ROOT/_pythonization/_tfile.py#L13

**Comment by skkwan** [comment]
Hi @vepadulano, thanks for the comments; I've changed the indenting and used the pythondoc macro.

**Comment by skkwan** [comment]
Oops, something strange happened while trying to fix a code formatting issue -- fixing now

**Comment by skkwan** [comment]
Deleting this PR and opening a new one -- apologies for the noise

**Pull request closed without merging**